### PR TITLE
Add FluentAssertions to project wizard

### PR DESCRIPTION
**Pull request by tzongithub**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [X] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Performance improvement
- [ ] Refactoring (so no functional change)
- [ ] Other (docs, build config, etc)

## Checklist:

- [ ] I've added tests for my code. (most of the time mandatory)
- [X] I have added an entry to the changelog. (mandatory)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: ProjectTemplateWizard/WizardImplementation.cs [hunk 1]
```diff
@@ -110,6 +110,7 @@ public void RunFinished()
                     // Add custom parameters.
                     replacementsDictionary.Add("$dotnetframework$", _inputDialog.DotNetFramework);
                     replacementsDictionary.Add("$unittestframework$", _inputDialog.UnitTestFramework);
+                    replacementsDictionary.Add("$fluentassertionsincluded$", _inputDialog.FluentAssertionsIncluded.ToString());
```
Comment (SabotageAndi) [inline ProjectTemplateWizard/WizardImplementation.cs:113]: will this always be in english or in the language of the user?

**Review by SabotageAndi — COMMENTED** [review]
Could you add a screenshot for the dialog page?

File: ProjectTemplateWizard/WizardImplementation.cs (hunk 1 above)
Comment (tzongithub) [reply ProjectTemplateWizard/WizardImplementation.cs:113]: image

I think this will be always in English.

Comment (tzongithub) [reply ProjectTemplateWizard/WizardImplementation.cs:113]: > 
> 
> will this always be in english or in the language of the user?

You're right, I should use the invariant culture explicitly.

**Review by SabotageAndi — APPROVED** [review]

**Pull request merged**